public/styles/devices.less

```diff
@@ -61,3 +65,85 @@
   margin-top: 20px;
   text-align: right;
 }
+
+.add-device {
+  position: absolute;
+  right: 5%;
+  bottom: 0;
+}
+
+.add-device button {
```

Comment:
This button change has some relation with the bug I reported to you (overlapping with device cards)? 

Independently, as this commit is getting bigger I advise you to break it into others.